### ISSUE TITLE
how to shorten HTTPS urls（Ver 1.5.1）？

### ISSUE THREAD

**Issue by eallion** [cydrobolt/polr#213]
im installing version 1.5.1
how to shorten HTTPS urls

i change “createurl.php”,but i failed

thanks very much

**Comment by cydrobolt** [comment]
Did you try simply putting an HTTPS URL into the input form on the main page? Did you receive an error?

**Comment by eallion** [comment]
no, i mean that....  i want to get a  HTTPS URL  result

**Comment by cydrobolt** [comment]
This feature is available in 2.0.0. See #117 
If you are unable to use 2.0.0, you can try editing `durl.js` with this

```
function y(a){return btoa(a)}function z(a){return atob(a)}function t(a){y(z(y(a)))}$(function(){function a(a){return z(a)}var b=$("#j").val(),c=a(b),d=$("#k").val(),e=a(d),f=a("aHR0cHM6Ly8="),g=z("Lw==");$("#i").val(f+e+g+c)});
```

**Comment by eallion** [comment]
it works, thanks very much!
